.prettierignore

```diff
@@ -5,3 +5,5 @@
 **/protos
 **/coverage
 .next
+**/.scripts
+**/.data
```

Comment:
I use these usually for debugging. Let me know if I should find another way to ignoring these locally.

Reply:
```suggestion
**/.data
```